internal/ops/store_op.go

```diff
@@ -1,22 +1,25 @@
 package ops
 
 import (
+	"net/http"
+
 	"github.com/dicedb/dice/internal/cmd"
 	"github.com/dicedb/dice/internal/comm"
 )
 
 type StoreOp struct {
-	SeqID     int16         // SeqID is the sequence id of the operation within a single request (optional, may be used for ordering)
-	RequestID int32         // RequestID identifies the request that this StoreOp belongs to
-	Cmd       *cmd.RedisCmd // Cmd is the atomic Store command (e.g., GET, SET)
-	ShardID   int           // ShardID of the shard on which the Store command will be executed
-	WorkerID  string        // WorkerID is the ID of the worker that sent this Store operation
-	Client    *comm.Client  // Client that sent this Store operation. TODO: This can potentially replace the WorkerID in the future
-	HTTPOp    bool          // HTTPOp is true if this Store operation is a HTTP operation
+	SeqID                int16               // SeqID is the sequence id of the operation within a single request (optional, may be used for ordering)
+	RequestID            uint32              // RequestID identifies the request that this StoreOp belongs to
+	Cmd                  *cmd.RedisCmd       // Cmd is the atomic Store command (e.g., GET, SET)
+	ShardID              int                 // ShardID of the shard on which the Store command will be executed
+	WorkerID             string              // WorkerID is the ID of the worker that sent this Store operation
+	Client               *comm.Client        // Client that sent this Store operation. TODO: This can potentially replace the WorkerID in the future
+	HTTPOp               bool                // HTTPOp is true if this Store operation is a HTTP operation
```

Comment:
@lucifercr07 we will need to refactor this as we move to multi-threading.
The shard instance will not have reference to HTTP request/response after the re-factoring.

cc @pratikpandey21

Reply:
I don't see any change in Ashwin's multithreading MR to `type StoreOp`. 
Are you talking about later? @soumya-codes

Reply:
Yes yes! 
We need these changes before the worker changes are checked in.
This was just FYI.

Just wanted Prashant to know about this.

Reply:
Cool.